spec/classes/config_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'spec_helper'
```

Comment:
I have moved away from testing all private classes. This is a nice idea, but with the current development practices it simply doesn't make sense. PDK has some stupid defaults.